specs/RuntimeSpec/Actions.md

```diff
@@ -91,13 +114,19 @@ pub struct TransferAction {
 }
 ```
 
-## StakeAction
-
 _Requirements:_
 
-- _`tx.signer_id` to be equal to `receiver_id`_
 - _`tx.public_key` to be `AccessKeyPermission::FullAccess` for the `singer_id`_
 
+**Outcome**:
+- transfers amount specified in `deposit` from `tx.signer` to a `tx.receiver_id` account
+
+**Errors**:
+- If the deposit amount plus the existing amount on the receiver account exceeds `u128::MAX`,
+a `StorageInconsistentState("Account balance integer overflow")` error will be returned.
```

Comment:
I think you can take a look at `verifiers.rs` and tests to see what errors are thrown during transaction validation.

There are some balance overflow, but it's not inconsistent state error.

There are also execution time errors, such as `AccountExists` that are thrown in `actions.rs`

Reply:
https://github.com/nearprotocol/nearcore/blob/1c1c019613e4ccb47ea6dc5a591f913af8022998/runtime/runtime/src/actions.rs#L293

Reply:
That's mostly only possible for challenges. We wrapped most errors in order to make sure when the challenge on malicious state transition arrives (where the previous root is also malicious), we don't crash the node.